packages/cosmic-swingset/src/chain-main.js

```diff
@@ -455,6 +462,7 @@ export default async function main(
 
     const slogSender = await makeSlogSender({
       stateDir: stateDBDir,
+      console: anylogger('slog-sender'),
```

Comment:
I don't remember what `makeSlogSender` does when using the subprocess option, but I suspect it doesn't route log calls through this console.